bindings/python/src/session.cpp

```diff
@@ -748,6 +748,7 @@ void bind_session()
         )
 #endif
         .def("post_torrent_updates", allow_threads(&lt::session::post_torrent_updates))
+        .def("post_session_stats", allow_threads(&lt::session::post_session_stats))
```

Comment:
Yea, no problem, I'll do that.

I was also looking at updating simple_client.py to include this function since it currently doesn't function with deprecated functions off.  In addition to the test.py update, would updating simple_client.py to use non-deprecated API be acceptable too?  Should we create a separate simple_client.py that continues to use deprecated functions?